src/Microsoft.Health.Dicom.Blob/Registration/DicomServerBuilderBlobRegistrationExtensions.cs

```diff
@@ -37,6 +39,25 @@ public static IDicomServerBuilder AddBlobDataStores(this IDicomServerBuilder ser
             .AddOptions<BlobOperationOptions>()
             .Bind(blobConfig.GetSection(nameof(BlobServiceClientOptions.Operations)));
 
+        FeatureConfiguration featureConfiguration = new FeatureConfiguration();
+        configuration.GetSection("DicomServer").GetSection("Features").Bind(featureConfiguration);
+        if (featureConfiguration.EnableExternalStore)
```

Comment:
Currently with external store enabled, `DcmHealthCheck` still runs against the internal blob store:

```
info: Azure.Core[1]
      Request [89d96890-0e0d-4d3f-a66a-3d2b7c3b63df] GET http://127.0.0.1:10000/devstoreaccount1/dicomwebcontainer?restype=container
```

Do we expect the health check to run against the external store in this case, or not run?

Reply:
Good point. I will condition it